roles/test_operator/tasks/collect-logs.yaml

```diff
@@ -37,9 +37,13 @@
     loop_var: pvc
     index_var: index
 
+- name: Set log pod fact to process omitted vars
+  ansible.builtin.set_fact:
+    _test_operator_log_pod: "{{ cifmw_test_operator_log_pod_definition }}"
```

Comment:
@karelyatin any reason why you don't apply `|to_nice_yaml` here in set fact and then you call the fact and use filter twice?
In that way, there is no reason to set fact.

Reply:
@danpawlik hmm i forgot to update the commit message when i revised the PR to get it working. The reason was mainly what is mentioned in the task name here, may be not clear enough.
Without this approach dumped yaml is not usable as contains omit_* string for all the vars used with default(omit) filter in default https://github.com/openstack-k8s-operators/ci-framework/blob/5f5393feade5031b54cd30589e2f964e44f54cd5/roles/test_operator/defaults/main.yml#L46-L61